send-transaction-service/src/send_transaction_service.rs

```diff
@@ -181,6 +184,150 @@ where
     }
 }
 
+/// Metrics of the send-transaction-service.
+#[derive(Default)]
+struct SendTransactionServiceStats {
+    /// Count of the received transactions
+    received_transactions: AtomicU64,
+
+    /// Count of the received duplicate transactions
+    received_duplicate_transactions: AtomicU64,
+
+    /// Count of transactions sent in batch
+    sent_transactions: AtomicU64,
+
+    /// Count of transactions not being added to retry queue
+    /// due to queue size limit
+    retry_queue_overflow: AtomicU64,
+
+    /// retry queue size
+    retry_queue_size: AtomicU64,
+
+    /// The count of batches
+    send_count: AtomicU64,
```

Comment:
It is not clear to me from the variable/metric name or the doc whether this is the number of batches sent, or number of transactions in batches.

Reply:
The doc should maybe also answer the question: does this include leader fanout?

Reply:
These are only send transaction calls -- not the count of transactions sent -- we already have sent_transactions. I updated the doc a little.